arkouda/numeric.py

```diff
@@ -2001,3 +2006,95 @@ def value_counts(
     (array([0, 2, 4]), array([3, 2, 1]))
     """
     return GroupBy(pda).count()
+
+
+@typechecked
+def clip(
+    pda: pdarray,
+    lo: Union[numeric_scalars, pdarray],
+    hi: Union[numeric_scalars, pdarray],
+) -> pdarray:
+    """
+    clip (limit) the values in an array to a given range [lo,hi]
+
+    Given an array a, values outside the range are clipped to the
+    range edges, such that all elements lie in the range.
+
+    There is no check on lo < hi, and if lo > hi, the corresponding
```

Comment:
I found this sentence confusing.  Perhaps something like "There is no check to enforce lo < hi, and if lo > hi...."